src/gcpsample_client/publishing.py

```diff
@@ -42,21 +42,6 @@
 video_spec = DigitalVideoSpecification.create()
 video_spec.displayNameTrait().setName("Ginos Home Video")
 # NOTE: It is critical to always use the working_ref from now on.
-working_ref = manager.preflight(entity_ref, video_spec, context)
-
-# We then check if the manager can tell us where to save the file.
-if ResolvesFutureEntitiesTrait.isImbuedTo(policy):
```

Comment:
This part is only if you want the manager to determine some aspects of the publish. A common example of this is a host asking the manager for the `LocatableContent` trait `location` to determine where to write a file, eg: during a render.